scripts/src/routes/offers.ts

```diff
@@ -15,8 +16,8 @@ router.get("/", async (req, res, next) => {
 		res.status(result.code || 200).send(result.data);
 	} catch (err) {
 		return res.status(500).send({
+			error: "Server Error",
```

Comment:
don't invest on these 500 errors. We should have a generic error handler that knows how to serialize exceptions thrown from the code.

Reply:
yeah, just fixing linting errors...